dist/smoothscroll.js

```diff
@@ -22,6 +22,9 @@
     return new RegExp(userAgentPatterns.join('|')).test(userAgent);
   }
 
+  var ALLOWED_BEHAVIOR_VALUES = [undefined, 'auto', 'instant', 'smooth'];
```

Comment:
https://drafts.csswg.org/cssom-view/#dictdef-scrollintoviewoptions

It is better to add the option "center".